remap_in:  [0,  1,   2,   3,   4,   5,   8,  9, 13]
remap_out: [50, 150, 250, 250, 250, 250, 250, 150, 250]
invalid_value: 0
